google/cloud/bigquery_storage_v1/writer.py

```diff
@@ -106,154 +106,48 @@ def __init__(
                 Extra headers to include when sending the streaming request.
         """
         self._client = client
-        self._closing = threading.Lock()
         self._closed = False
         self._close_callbacks = []
-        self._futures_queue = queue.Queue()
         self._metadata = metadata
+        self._thread_lock = threading.RLock()
+        self._closed_connection = {}
+
+        # Max number of connections in the process of closing.
+        self._max_closing_connections = 1
```

Comment:
This doesn't appear to be used, can probably remove it (and comment).

Reply:
Thanks a lot for catching this!

Reply:
removed